src/Kripke/Arch/Population.h

```diff
@@ -216,9 +216,9 @@ struct Policy_Population<ArchLayoutT<ArchT_CUDA, LayoutT_DGZ>>{
   using ExecPolicy =
     KernelPolicy<
       CudaKernel<
-        Tile<2, tile_fixed<32>, cuda_block_x_loop, 
-          For<1, cuda_thread_x_direct,
-            For<0, cuda_thread_y_direct,
+        For<2, cuda_block_x_loop,
+          For<1, cuda_thread_x_loop,
```

Comment:
Why did you remove the tiling in these policies? @ajkunen should weigh in on this.

Reply:
@rhornung67 that was my suggestion

Reply:
To ask you to weigh in? Seems circular.... :-)